app/models/agents/twitter_stream_agent.rb

```diff
@@ -70,6 +81,7 @@ def working?
     def default_options
       {
         'filters' => %w[keyword1 keyword2],
+        'include_retweets' => "false",
```

Comment:
I think we should keep the previous default of `true` here. The most common use case I know of the Agent is to check for spikes of mentions for a hashtag/word where retweets are an indicator as well.

Reply:
I took [the following lines](https://github.com/huginn/huginn/blob/master/app/models/agents/twitter_stream_agent.rb#L225-L226) to indicate the default was silently dropped retweets. I'm pretty new to Ruby, so there's a good chance I'm wrong, can you confirm? 

```
        if status["retweeted_status"].present? && status["retweeted_status"].is_a?(Hash)
          return
```